rosa/cli.py

```diff
@@ -51,14 +54,21 @@ def is_logged_in(aws_region=None, allowed_commands=None):
 
 
 def execute_command(command):
+    def _wait_for_command_execution(_command):
```

Comment:
Update:
The TimeoutSampler works fine, but doesn't fix the problem of being hold forever by the rosa cli's interactive mode, also when timeout exceeds (the sampler doesn't return while the cli is waiting for user input).